src/app/pages/404/404.hbs

```diff
@@ -1,5 +1,11 @@
-<h1>Not found</h1>
-<p>
-    Kudos on your desire to explore! Unfortunately, we don't have a page to go
-    with that particular location.
-</p>
+<style scoped>@import url(/app/pages/404/404.css);</style>
+<div class="not-found page">
```

Comment:
we have .page and .boxed classes in our main.css file. i just added these divs in this page for consistency since all the other pages have this. I can remove it if you think its best.